### PR TITLE
Add .NET developer page

### PR DESCRIPTION
**Pull request by Solexplorer**
## Description
Added .NET developer page with all the tools available for .NET developers

## Related Issue

#267

## Checklist:

- [X] I have read the contributing guidelines in the project README.
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/net/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (samajammin) [inline docs/net/index.md]: Let's go with `dot-net` for the directory.

File: docs/net/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+sidebarDepth: 0
+---
+
+# Ethereum for .NET Developers
+
+<div class="featured">Learn how to develop for Ethereum using .NET-based projects and tooling</div><br>
+
+Use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
+
+<img src="https://i.imgur.com/onFLM4L.png" width="100%" />
```
Comment (samajammin) [inline docs/net/index.md]: Did you create this image yourself? It displays massive on desktop & the text is hard to read:

Image 2019-11-05 at 3 43 35 PM

Let's remove this unless you can provide a cleaner image of similar dimensions to /python & /java (~240px height):
Image 2019-11-05 at 3 45 00 PM

File: docs/net/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- [Writing Ethereum Transactions to SQL Database Using Nethereum](https://medium.com/coinmonks/writing-ethereum-transactions-to-sql-database-using-nethereum-fd94e0e4fa36)
+- [See how to easily deploy Ethereum smart contracts using C# and VisualStudio](https://koukia.ca/deploy-ethereum-smart-contracts-using-c-and-visualstudio-5be188ae928c)
+- [A .NET Developer’s Workflow for Creating and Calling Ethereum Smart Contracts](https://medium.com/coinmonks/a-net-developers-workflow-for-creating-and-calling-ethereum-smart-contracts-44714f191db2)
+
+## Intermediate Articles
+
+- [Quick introduction to smart contracts integration with Nethereum](https://nethereum.readthedocs.io/en/latest/Nethereum.Workbooks/docs/nethereum-gettingstarted-smartcontracts/)
+- [Interfacing .NET and Ethereum Blockchain Smart Contracts with Nethereum](https://medium.com/singapore-blockchain-dapps/interfacing-net-and-ethereum-blockchain-smart-contracts-with-nethereum-7b46b71f2dae)
+- [https://tech-mint.com/create-asp-net-core-web-api-for-ethereum-dapps/](https://tech-mint.com/create-asp-net-core-web-api-for-ethereum-dapps/)
```
Comment (samajammin) [inline docs/net/index.md]: Please provide description text here vs. the raw URL

**Comment by samajammin** [comment]
@CPSTL could you please help out with a review on this? My .NET knowledge is limited 😄

File: docs/net/index.md (hunk 3 above)
Comment (Solexplorer) [reply docs/net/index.md]: Oopsie, missed this

**Comment by Solexplorer** [comment]
Hey @samajammin,

I have implemented the changes requested.

Thanks!

**Pull request merged**